### PR TITLE
Update 01-Overview.md

### PR DESCRIPTION
**Pull request by okwme**
tiny changes

### CODE REVIEW

File: hs-abci-docs/nameservice/tutorial/Foundations/02-Effects.md [hunk 1]
```diff
@@ -87,12 +87,12 @@ type StoreEffs =
   ]
 ~~~
 
- They are used to interpret `TxEffects` depending on what context you're in, e.g. while executing a `delierTx` versus `checkTx` message, or a `query`. Some effects are tagged with a promoted value of type `Scope`, i.e. `'Consensus` and `'QueryAndMempool`. This is because your application will keep multiple connections to the database that are used in different situations. For example, since writing to the state at a previous blocktimes is impossible, we disallow writing to the database in such instances.
+They are used to interpret `TxEffects` depending on what context you're in, e.g. while executing a `delierTx` versus `checkTx` message, or a `query`. Some effects are tagged with a promoted value of type `Scope`, i.e. `'Consensus` and `'QueryAndMempool`. This is because your application will keep multiple connections to the database that are used in different situations. For example, since writing to the state at a previous blocktime is impossible, we disallow writing to the database in such instances.
```
Comment (martyall) [inline hs-abci-docs/nameservice/tutorial/Foundations/02-Effects.md:90]: looks like there's another typo here 

executing a `delierTx ~> deliverTx` versus

File: hs-abci-docs/nameservice/tutorial/Foundations/02-Effects.md [hunk 2]
```diff
@@ -106,4 +106,4 @@ There is a type alias in the SDK called `BaseAppEffs` with the definition
 type BaseApp core = BaseEffs :& StoreEffs :& core
 ~~~
 
-It sits at the bottom of any applications effects list and ultimately everything is interpreted through these effects before being run, e.g. `TxEffs` and `QueryEffs`. This effects list is pretty much the only place where the application developer needs to decide how the interpretation should be done. There are two options in the SDK, using `PureCoreEffs` or `CoreEffs` depending on whether you want to rely on an external or in-memory database.
\ No newline at end of file
+It sits at the bottom of any applications effects list and ultimately everything is interpreted through these effects before being run, e.g. `TxEffs` and `QueryEffs`. This effects list is pretty much the only place where the application developer needs to decide how the interpretation should be done. There are two options in the SDK, using `PureCoreEffs` or `CoreEffs` depending on whether you want to rely on an external or in-memory database.
```
Comment (martyall) [inline hs-abci-docs/nameservice/tutorial/Foundations/02-Effects.md:109]: bottom of any `applications ~> application's` effects list

**Comment by martyall** [comment]
Thanks for doing this, appreciate the work! Seems like you are still reading so just ping me here when you're done.

**Comment by okwme** [comment]
Accidentally deleted my working repo 🤦
Luckily this PR made it easy to move them into new one #231

**Pull request closed without merging**